client/js/jsonedit.js

```diff
@@ -1356,14 +1356,27 @@ function jeLoggingRoutineEnd(variables, collections) {
   --jeLoggingDepth;
   if(!jeLoggingDepth) {
     $('#jeLog').innerHTML = jeLoggingHTML + '</div></div>';
-    var expanders = document.getElementsByClassName('jeExpander');
-    var i;
+    // Make it so clicking on the arrows expands the subtree
+    const expanders = document.getElementsByClassName('jeExpander');
```

Comment:
`$a('.jeExpander')` should do the same. Yours might be slightly faster so just FYI.

Reply:
Do you mean `$('.jeExpander')`?

Reply:
No. That only returns the first one. `a` stands for `all` (or `array`) and returns all of them in an array.

Reply:
Good to know. Thanks. (Where is this documented in, say, the w3schools pages?)

Reply:
Nope: https://github.com/ArnoldSmith86/virtualtabletop/blob/main/client/js/domhelpers.js#L1-L7

Reply:
Oh. I never looked at that file. Guess I should :)